fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2685,6 +2685,24 @@ public static boolean isNotCloudMode() {
 
     @ConfField(mutable = true, masterOnly = false)
     public static String security_checker_class_name = "";
+    public static int history_cloud_warm_up_job_keep_max_second = 7 * 24 * 3600;
```

Comment:
miss  @ConfField(?
BTW, we should use global session variable as far as possible